File: CSharp/Account.cs
using System;

namespace CSharp
{
    class Account
    {
        private int id;
        private string name;
        private string document;
        private string email;
        private string password;

        public Account(int id, string name, string document, string email, string password)
        {
            this.id = id;
            this.name = name;
            this.document = document;
            this.email = email;
            this.password = password;
        }

        public override string ToString()
        {
            return $"ID: {id}, Name: {name}, Document: {document}, Email: {email}, Password: {password}";
        }
    }
}

File: Java/src/Account.java
public class Account {
    private Integer id;
    private String name;
    private String document;
    private String email;
    private String password;

    public Account(Integer id, String name, String document, String email, String password) {
        this.id = id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }

    @Override
    public String toString() {
        return "Account{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", document='" + document + '\'' +
                ", email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}
